src/hmpps/utils/utils.ts

```diff
@@ -37,3 +37,9 @@ export const personProfileName = (person: { firstName: string; lastName: string
 export const personDateOfBirth = (dateOfBirth: string): string => dayMonthYearForwardSlashSeparator(dateOfBirth)
 
 export const personStatus = (status: string): string => sentenceCase(status)
+
+export const hmppsFormatDate = (dateString: string, pattern: string): string => {
```

Comment:
This might be a thing we need to consider in the future, there are two date formats/ the ability to give any pattern! One for the person's date of birth https://github.com/ministryofjustice/hmpps-court-cases-release-dates-design/blob/c01defb5b52ca1ebcabe9db4d6a971b1e58ad8b7/src/hmpps/utils/utils.ts#L37 and the other for a calculated date, guessing its contextual as to which date format to use rather than 1 pattern to rule them all!